client/jetpack-connect/user-type/style.scss

```diff
@@ -50,6 +50,15 @@
 	&:focus {
 		box-shadow: inset 0 0 0 2px var( --color-accent );
 	}
+
+	&:first-of-type {
```

Comment:
Could we use `:first-child` and `:last-child` instead, for consistency reasons? Seems like that's what we're already using for the site type component:

https://github.com/Automattic/wp-calypso/blob/e64001e08c5c7d24be02021b4609e171a84ab2ae/client/signup/steps/site-type/style.scss#L31

Reply:
Good catch! I've updated it on https://github.com/Automattic/wp-calypso/pull/31712/commits/536284fb5a4246a60380ed92999d0c4cc841c1d2